src/useWrappedRefWithWarning.js

```diff
@@ -0,0 +1,21 @@
+import invariant from 'invariant';
+import { useCallback } from 'react';
+import useMergedRefs from '@restart/hooks/useMergedRefs';
+
+export default function useWrappedRefWithWarning(ref, componentName) {
+  if (!__DEV__) return ref;
```

Comment:
can we not test for `process.env.NODE_ENV === "test"`?
So we can omit this __DEV__ variable.

Reply:
this is actually shorthand for `process.env.NODE_ENV === 'production'` Babel replaces instances of `__DEV__` with the process.env check via the `dev-expression` plugin, as well as rewriting some of the invariant or warning calls so they can be dead-code eliminated when applicable